frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -24,21 +16,21 @@ import { convertAmountToUsage, convertUsageToAmount, summarizeUsage } from './bi
 import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
 import { capitalizeFirstLetter } from 'lib/utils'
 import { useResizeBreakpoints } from 'lib/hooks/useResizeObserver'
-import { BillingHero } from './BillingHero'
 import { IconDelete, IconEdit } from 'lib/components/icons'
+import { PlanTable } from './PlanTable'
+import { BillingHero } from './BillingHero'
 
 export type BillingV2Props = {
     redirectPath?: string
-    compact?: boolean
+    showCurrentUsage?: boolean
```

Comment:
I added this new table to the onboarding panel (both v2 control and test), but hiding the current usage through this toggle (future step, we should just break this down into smaller components as you suggested)